### ISSUE TITLE
MANORM_beds searching for beds that have already been gzipped

### ISSUE THREAD

**Issue by slsevilla** [RBL-NCI/iCLIP#120]
Ran into an issue with rule MANORM_beds

error location: /data/RBL_NCI/Wolin/mES_fclip_1_YL_011622_v2.0/log/20220705_1143/21a_MANORM_beds.43115030.sp=YKO_fclip.err

error snippet:
```
$6=="+"' /data/RBL_NCI/Wolin/mES_fclip_2_YL_020922_v2.0/03_peaks/01_bed/untagged_fclip_ALLreadPeaks.bed
+ awk '$6=="-"' /data/RBL_NCI/Wolin/mES_fclip_2_YL_020922_v2.0/03_peaks/01_bed/untagged_fclip_ALLreadPeaks.bed
awk: fatal: cannot open file `/data/RBL_NCI/Wolin/mES_fclip_2_YL_020922_v2.0/03_peaks/01_bed/untagged_fclip_ALLreadPeaks.bed' for reading (No such file or directory)
```

**Comment by slsevilla** [comment]
Rule MANORM_beds input is 
```
rules.create_beds_safs.output.bed_all
```

Rule create_beds_safs output is:
```
bed_all = (join(out_dir,'03_peaks','01_bed','{sp}_ALLreadPeaks.bed')),
```

However, following this rule bgzip_beds gzips the bed files, with the output:
```
bed_all = join(out_dir,'03_peaks','01_bed','{sp}_ALLreadPeaks.bed.gz'),
```

This means the input to MANORMS_beds should be bgzip_beds, not create_beds_safs. Depending on which rule is deployed at which time, some of these rules complete, presumably because the gzip rule has not completed yet. Previous versions did not have this error as all files were kept.

ACTION: changing the input of MANORMS_beds, and adding zcat to shell command.
``` 
#from 
rules.create_beds_safs.output.bed_all

#to
rules.bgzip_beds.output.bed_all
```

**Comment by slsevilla** [comment]
Because re-runs of the pipeline would require the original bed files to complete, zipped bedfiles are created and marked as required, while unzipped bedfiles are marked as temp. Will implement this update.

**Comment by slsevilla** [comment]
issue fixed with commit 564dc1bb71d52bb8b09c112c041e87d3a4186c3c
